agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3091,6 +3091,12 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
                                   config.backhaul_wireless_iface.c_str(),
                                   message::IFACE_NAME_LENGTH);
 
+        auto tuple_supported_channels = bh_enable->supported_channels_list(0);
+        std::copy_n(hostap_params.supported_channels, message::SUPPORTED_CHANNELS_LENGTH,
+                    &std::get<1>(tuple_supported_channels));
+
+        bh_enable->ruid() = network_utils::mac_from_string(config.radio_identifier);
```

Comment:
Important - see #691 - this will need to change to the radio mac if #431 will be merged before this change (and it probably will)

Reply:
I've removed the ruid field